### DISCUSSION TITLE
Procedura di aggiornamento

### DISCUSSION THREAD

**Discussion by rodolfo024** [iisgiua/giuaschool#439] [Richieste di aiuto]
Come bisogna procedere per aggiornare con il fix degli scrutini intermedi? Da Sistema -> Aggiornamenti non risultano aggiornamenti. Immagino bisogni procedere in modo "tradizionale" ma ricordo che l'ultima volta era diversa la procedura.

**Comment by trinko** [comment]
Per installare le modifiche successive al rilascio della versione 1.5.1 bisogna usare il metodo tradizionale: caricare il codice sul sito e poi eseguire l'installazione dalla pagina **http://nome_sito/install/index.php**

**Comment by ghost** [comment]
@rodolfo024 Buongiorno! Avrei bisogno di parlarti, in merito a questa richiesta. Posso parlarti in privato? Mi scrivi alla mia mail: padovesi.roberto@gmail.com? Grazie!
